bot/seasons/season.py

```diff
@@ -132,11 +134,18 @@ def greeting(self) -> str:
         """
         return f"New Season, {self.name_clean}!"
 
-    async def get_icon(self, avatar: bool = False) -> bytes:
+    @property
+    def cycle(self) -> bool:
+        """Return whether the icon attribute is a list and hence needs to be cycled through."""
+        return isinstance(self.icon, list)
```

Comment:
I think it'd be better to check if it's a `typing.Sequence` rather than locking it into being a list.